sql/src/main/java/io/crate/analyze/WindowDefinition.java

```diff
@@ -75,12 +75,24 @@ public WindowDefinition(List<Symbol> partitions,
         return partitions;
     }
 
+    public WindowFrameDefinition mapWindowFrameDefinitionBounds(Function<? super Symbol, ? extends Symbol> mapSymbolsFunction) {
+        FrameBoundDefinition startFrameBoundDef = windowFrameDefinition.start();
+        if (startFrameBoundDef.value() != null) {
```

Comment:
`FrameBoundDefinition.map` already takes care of the null handling. So we can remove the `!= null` here.
(And also avoid the `.start()` local variable assignment that is then not used in the next line here)